### PR TITLE
Add new flag to specify queues to avoid

### PR DESCRIPTION
**Pull request by rk1274**

### CODE REVIEW

File: jobqueue/scheduler/lsf.go [hunk 1]
```diff
@@ -365,41 +363,14 @@ func (s *lsf) initialize(ctx context.Context, config interface{}) error {
 			}
 			punishment := rank * weight
```
Comment (sb10) [inline jobqueue/scheduler/lsf.go:364]: This can now just be `ranking[queue] += rank * weight`?

File: jobqueue/scheduler/lsf.go [hunk 2]
```diff
@@ -321,13 +320,13 @@ func (s *lsf) initialize(ctx context.Context, config interface{}) error {
...
-		"max":        {15, 1},
-		"prio":       {4, 1},
+		"hosts":      {18, 1}, // weight, sort order
+		"max_user":   {10, 1},
+		"max":        {5, 1},
+		"prio":       {5, 0},
 		"chunk_size": {10000, 0},
-		"runlimit":   {2, 0},
-		"memlimit":   {2, 0},
+		"runlimit":   {5, 1},
```
Comment (sb10) [inline jobqueue/scheduler/lsf.go:328]: I'm a bit concerned that you had to change the sort order on 2 of these. Did it make logical sense to you that this was the correct thing to do?

File: jobqueue/scheduler/lsf.go [hunk 3]
```diff
@@ -628,24 +599,19 @@ func (s *lsf) busy(ctx context.Context) bool {
 // contains a scheduler_queue value, returns that instead.
 // *** globalMax option and associated code may be removed if we never have a
```
Comment (sb10) [inline jobqueue/scheduler/lsf.go:600]: remove this and and other mention of globalMax in all code

**Review by sb10 — CHANGES_REQUESTED** [review]

File: jobqueue/scheduler/lsf.go (hunk 2 above)
Comment (rk1274) [reply jobqueue/scheduler/lsf.go:328]: Changing the runlimit to be {5, 1} means a higher runlimit is favoured, the original way meant that the gpu queues were ranked much higher than wanted, and the new way puts those gpu-queues much nearer the end.

The change for prio to {4, 0} means normal, long, parallel and week all rank much better as they all have lower prios. Without this, week ranks worse than several queues.

**Review by sb10 — APPROVED** [review]

**Pull request merged**